.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "4"
```

Comment:
Yeah I made a diff comment so it should look like the change with the -  of one line for removal in red and the + in green for the add